network/src/commonTest/kotlin/com/wire/kalium/api/TestSessionManagerV0.kt

```diff
@@ -22,6 +22,14 @@ class TestSessionManagerV0 : SessionManager {
             newRefreshTokenDTO?.value ?: session.refreshToken
         )
 
+    override suspend fun beforeTokenUpdate() {
+        TODO("Not yet implemented")
+    }
+
+    override fun afterTokenUpdate() {
+        TODO("Not yet implemented")
+    }
+
```

Comment:
Are we planning to test this at some point?

Reply:
i just removed it and moved the whole logic to SessionManager